ros_message/src/time.rs

```diff
@@ -306,8 +326,24 @@ impl ops::Neg for Duration {
 impl From<time::Duration> for Duration {
     fn from(std_duration: time::Duration) -> Self {
         Duration {
-            sec: std_duration.as_secs() as i32,
+            sec: std_duration.as_secs().try_into().unwrap(),
```

Comment:
`expect("Durations longer than 68 years are not supported by the ROS time format")`